rafs/src/metadata/layout/v6.rs

```diff
@@ -174,22 +181,41 @@ impl RafsV6SuperBlock {
             )));
         }
 
-        if self.s_inos == 0 {
+        if u64::from_le(self.s_inos) == 0 {
```

Comment:
no need to convert endianness when comparing with 0

Reply:
fixed